scripts.js

```diff
@@ -11,35 +14,98 @@ function createJSObject(json) {
 
     dict = new Object();
 
-    for (let i = 0; i < json.rounds.length; i++) {
-        let matches = json.rounds[i].matches;
+    for (let i = 0; i < json.matches.length; i++) {
 
+        let games = json.matches[i];
 
-        for (let j = 0; j < matches.length; j++) {
-            let curMatch = matches[j];
-            let homeTeam = curMatch.team1;
-            let awayTeam = curMatch.team2;
+        
+        let homeTeam = games.team1;
+        let awayTeam = games.team2;
 
+        if (!(homeTeam in dict)) {
+            dict[homeTeam] = [];
+        }
 
-            if (!(homeTeam in dict)) {
-                dict[homeTeam] = [];
-            }
+        if (!(awayTeam in dict)) {
+            dict[awayTeam] = [];
+        }
 
-            if (!(awayTeam in dict)) {
-                dict[awayTeam] = [];
-            }
+        dict[homeTeam].push(games);
+        dict[awayTeam].push(games);
 
-            dict[homeTeam].push(curMatch);
-            dict[awayTeam].push(curMatch);
-        }
+    
     }
-
     console.log(dict);
 }
 
-
 // load JSON data; then proceed
-getJSON('https://raw.githubusercontent.com/openfootball/football.json/master/2015-16/en.1.json').then(data => {
+getJSON('https://raw.githubusercontent.com/openfootball/football.json/master/2020-21/en.1.json').then(data => {
     eplMatchlist = data;
-}
-);
\ No newline at end of file
+})
+
+$(document).ready(function () {
+    score_card_prefab = document.getElementById('card_start').cloneNode(true);
+
+    var parent = document.getElementById('score_table');
+    parent.innerHTML = '';
+
+    $('a').click(function () {
+
+        var curId = $(this).attr('id');
+
+        parent.innerHTML = '';
+        
+
+        //going through the list of elements by id
+        for (let i = 0; i < 10; i++) {
+
+            //cloning the prefab element
+            let score_card = score_card_prefab.cloneNode(true);
+
+            // getting names of home and away teams along with match date
+            let homeNameText = dict[curId][i].team1;
+            let awayNameText = dict[curId][i].team2;
+            let gameDate= dict[curId][i].date;
+            
+            // removing whitepace from home and away teams to match img id
+            var trimHome = homeNameText.replace(/\s/g, '');
```

Comment:
let not var in a for loop
```suggestion
            let trimHome = homeNameText.replace(/\s/g, '');
```